src/renderers/shaders/ShaderLib.js

```diff
@@ -93,7 +93,7 @@ const ShaderLib = {
 				emissive: { value: /*@__PURE__*/ new Color( 0x000000 ) },
 				roughness: { value: 1.0 },
 				metalness: { value: 0.0 },
-				envMapIntensity: { value: 1 } // temporary
```

Comment:
I market this as 'temporary' in the original MeshStandardMaterial PR (#7381) because I felt the environmant map intensity should be a property of the environment map, not the material.

But it has turned out not to be temporary after all...